### PR TITLE
Added Adwords Remarketing Tag Insertion

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/2211
@pdpinch 

### Note:
set .env variable ```ADWORDS_CONVERSION_ID``` to make it work

### CODE REVIEW

File: ui/templates/base.html [hunk 1]
```diff
@@ -58,6 +58,19 @@
         smartlook('init', '{{ APIKEYS.SMARTLOOK }}');
     </script>
     {% endif %}
+    <script type="text/javascript">
+      /* <![CDATA[ */
+      var google_conversion_id = 935224753;
```
Comment (rhysyngsun) [inline ui/templates/base.html]: This value should probably be in an environment variable in case it changes.

File: ui/templates/base.html [hunk 2]
```diff
@@ -58,6 +58,19 @@
         smartlook('init', '{{ APIKEYS.SMARTLOOK }}');
     </script>
     {% endif %}
+    <script type="text/javascript">
```
Comment (rhysyngsun) [inline ui/templates/base.html]: I believe this template is applied to all pages, but the original ticket asks for it to only show up on 3. Is that going to be an issue?

Comment (amir-qayyum-khan) [inline ui/templates/base.html]: @rhysyngsun please see this thread and https://github.com/mitodl/micromasters/issues/2211#issuecomment-270929044

**Comment by amir-qayyum-khan** [comment]
Done @rhysyngsun

File: micromasters/settings.py [hunk 3]
```diff
@@ -437,6 +437,7 @@ def get_var(name, default):
 STATUS_TOKEN = get_var("STATUS_TOKEN", "")
 HEALTH_CHECK = ['CELERY', 'REDIS', 'POSTGRES', 'ELASTIC_SEARCH']
 
+ADWORDS_CONVERSION_ID = get_var("ADWORDS_CONVERSION_ID", "")
```
Comment (rhysyngsun) [inline micromasters/settings.py]: Should this get added to `app.json` as an optional var so it shows up in the Heroku settings dashboard?

**Comment by rhysyngsun** [comment]
@amir-qayyum-khan code looks good, feel free to merge after adding to app.json and it deploys correctly.

**Pull request merged**